src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaPlayerElement.cs

```diff
@@ -381,6 +381,10 @@ public void SetMediaPlayer(Windows.Media.Playback.MediaPlayer mediaPlayer)
 			MediaPlayer = mediaPlayer;
 		}
 
+		#region IsCompactOverlay Property
+
+		public bool IsCompactOverlay { get; set; }
```

Comment:
Is this part of the public API from MediaPlayerElement ?

Reply:
not part of the control: https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.mediaplayerelement?view=winrt-22621
and we arent using it, so i had it removed in my pr
this is probably from a bad merge/rebase

we have similar named properties on MediaTransportControls:
- IsCompact
- IsCompactOverlayEnabled
- IsCompactOverlayButtonVisible

Reply:
i remove the IsCompactOverlay from the MediaPlayerElement. And now using the IsCompact from MTC.
I left the feature as it is changing for CompactMode and just for wasm the PIP working.